app/controllers/coronavirus/announcements_controller.rb

```diff
@@ -11,21 +11,21 @@ def new
     def create
       @announcement = page.announcements.new(announcement_params)
       if @announcement.save && draft_updater.send
-        redirect_to coronavirus_page_path(page.slug), notice: "Announcement was successfully created."
+        redirect_to coronavirus_page_path(page.slug), notice: I18n.t("coronavirus.pages.announcements.new.success")
```

Comment:
I didn't move the announcements section in the locale file. It was always under "pages". I left it there because a page has announcements. But I don't think the pages controller and page model are related like that are they?

Also, I used `new.success` because I thought of it in terms of what the user is doing, i.e. adding a "new" announcement.